app/src/main/java/org/p2p/wallet/moonpay/api/MoonpayIpAddressResponse.kt

```diff
@@ -0,0 +1,12 @@
+package org.p2p.wallet.moonpay.api
+
+import com.google.gson.annotations.SerializedName
+
+class MoonpayIpAddressResponse(
+    @SerializedName("alpha3")
+    val alpha3: String,
+    @SerializedName("isAllowed")
+    val isAllowed: Boolean,
```

Comment:
isAllowed what action?

Reply:
Both `isAllowed` and `isBuyAllowed` will be used in next iteration for buy restriction for specific countries and I didn't really deep dive into that fields